dev/gulpfile.js

```diff
@@ -55,12 +68,12 @@ let pack = async (files) => {
   }
 };
 
-gulp.task('build', async function() {
+gulp.task('build', ['arcs-build'], async function() {
   await pack(sources.cdn);
 });
 
 const components = `${target}components/`;
-const arcs = `../arcs/`;
+const arcs = `../../arcs/`;
```

Comment:
I noticed this wasn't correct.. and it meant that the 'copy' target wasn't working, and hasn't been for some time. It still copies files once this is fixed, but I wonder if I should just delete the target?

Reply:
Perhaps we just haven't made sufficiently breaking changes to `strategyExplorer` or `suggestionsElement` in a while? Seems worth keeping target for now since we're still using both of those. @mmandlis may know more.

`suggestions-element.js` was last modified in `arcs` on Sep 19. The `arcs-cdn` version has seen a few mods by @sjmiles since then. There are some diffs. Hmm.  https://gist.github.com/shaper/85639f5c8b49b149dc415c48d429f0ac

Reply:
:/